nansat/tests/test_domain.py

```diff
@@ -425,13 +425,13 @@ def test_get_min_max_lon_lat(self):
         self.assertLess(result[2], result[3])
         self.assertEqual(result, (25.0, 34.980000000000004, 70.004000000000005, 72.0))
 
-    def test_get_pixelsize_meters(self):
+    def test_get_pixel_size_meters(self):
```

Comment:
this method is not testing the new property. Instead of calling ```get_pixel_size_meters``` in lines 430 and 434, you should call the new property as ```d.pixel_size```.